src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
 
 class PageObject {
+  constructor(url, header) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  commentButton = '[Publish comment] button';
+  constructor(url, header, slug) {
```

Comment:
The constructor of the `ArticlePage` class should only take a `url` and a `slug` parameter. Remove the `header` parameter. 🚀